### PR TITLE
Add Prettier support

### PR DESCRIPTION
**Pull request by lipis**
https://github.com/prettier/prettier

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,12 +34,12 @@
...
     "dist": "yarn clean && tsc",
-    "lint": "echo \"No linting specified\" && exit 0",
-    "preversion": "yarn lint && yarn dist && yarn test",
-    "version": "yarn dist",
     "postversion": "git push && git push --tags",
+    "prettier": "prettier --single-quote --trailing-comma --no-bracket-spacing  'src/**/*.ts'",
+    "preversion": "yarn lint && yarn dist && yarn test",
     "start": "yarn clean && concurrently \"tsc -w\" \"yarn watch\"",
     "test": "jasmine",
+    "version": "yarn dist",
```
Comment (lipis) [inline package.json]: I sorted them.. that's why they look messy :/

File: src/ts/core/WireAPIClient.ts [hunk 2]
```diff
@@ -8,28 +8,28 @@ import TeamAPI from '../team/TeamAPI';
...
 
 export default class WireAPIClient extends EventEmitter {
-  public auth: { api: AuthAPI } = {
-    api: undefined
+  public auth: {api: AuthAPI} = {
+    api: undefined,
   };
 
-  public client: { http: HttpClient, ws: WebSocketClient } = {
+  public client: {http: HttpClient; ws: WebSocketClient} = {
```
Comment (lipis) [inline src/ts/core/WireAPIClient.ts]: that's strange! :) `;`

**Comment by bennyn** [comment]
LPGTM. Looks **pretty** good to me. 🤣

**Comment by lipis** [comment]
The future! :)

**Pull request merged**